qt/widgets/instrumentview/test/InstrumentWidget/InstrumentWidgetTest.h

```diff
@@ -64,178 +64,213 @@ class InstrumentWidgetTest : public CxxTest::TestSuite {
   }
 
   void test_constructor() {
-    auto qtMock = makeQtDisplay();
-    auto glMock = makeGL();
-    auto instance =
-        construct("test_ws", makeDisplay(), qtMock.get(), glMock.get(), makeConnect(), makeMetaObject(), 22);
-
-    InstrumentActor &actor = instance.getInstrumentActor();
-    actor.initialize(true, true);
-    instance.initWidget(true, true);
-    instance.waitForThread();
+    for (const bool useLoadingThread : {true, false}) {
```

Comment:
I've not come across this before - how does it work? To my eye it looks like useLoadingThread is both true and false but I'm sure that can't be right.

Reply:
In Python the equivalent would be:
``` python
def test_constructor:
   for i in [True, False]:
      foo = construct(arg1, arg2, ..., i)
```